connectors/sources/sharepoint_server.py

```diff
@@ -295,26 +295,6 @@ async def get_site_pages_content(
             "_attachment": await self.convert_file_to_b64(source_file_name),
         }
 
-    def _get_retry_after(self, retry, exception):
```

Comment:
Why is the method deleted?

Reply:
In this PR we have remove the logic to raise the exception after 3 retries, instead of raising it now we are skipping that call. And to achieve this we need to add `break`  inside the condition `if retry > self.retry_count:`. 
to add `break` loop is required and that's why we have removed this method and directly use it in `_api_call` to skip the call.